app/views/workflows/_form.html.erb

```diff
@@ -26,48 +26,72 @@ Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
                                      :alt => "#{l(:button_check_all)}/#{l(:button_uncheck_all)}") %>
-          <%=h old_status.name %>
-        </td>
-        <% for new_status in @statuses -%>
-          <td align="center">
-            <%= check_box_tag "status[#{ old_status.id }][#{new_status.id}][]", name, workflows.detect {|w| w.old_status_id == old_status.id && w.new_status_id == new_status.id},
-            :class => "old-status-#{old_status.id} new-status-#{new_status.id}" %>
-          </td>
-        <% end -%>
-      </tr>
-    <% end %>
-  </tbody>
-</table>
+                  <%=l(:label_current_status)%>
+                </span>
+              </div>
+            </div>
+          </th>
+          <th colspan="<%= @statuses.length %>">
+            <div class="generic-table--sort-header-outer">
+              <div class="generic-table--sort-header" style="text-align: center;">
+                <span>
+                  <%=l(:label_new_statuses_allowed)%>
+                </span>
+              </div>
+            </div>
+          </th>
+        </tr>
+        <tr>
+          <td></td>
+          <% for new_status in @statuses %>
+            <td >
+              <%= link_to_function(icon_wrapper('icon-context icon-yes',"#{l(:button_check_all)}/#{l(:button_uncheck_all)}"), "toggleCheckboxesBySelector('table.transitions-#{name} input.new-status-#{new_status.id}')",
+                               :class => 'no-decoration-on-hover',
+                               :title => "#{l(:button_check_all)}/#{l(:button_uncheck_all)}",
+                               :alt => "#{l(:button_check_all)}/#{l(:button_uncheck_all)}") %>
+              <%=h new_status.name %>
+            </td>
+          <% end %>
+        </tr>
+      </thead>
+      <tbody>
+        <% for old_status in @statuses %>
+          <tr>
+            <td>
+              <%= link_to_function(icon_wrapper('icon-context icon-yes',"#{l(:button_check_all)}/#{l(:button_uncheck_all)}"), "toggleCheckboxesBySelector('table.transitions-#{name} input.old-status-#{old_status.id}')",
+                                         :class => 'no-decoration-on-hover',
+                                         :title => "#{l(:button_check_all)}/#{l(:button_uncheck_all)}",
+                                         :alt => "#{l(:button_check_all)}/#{l(:button_uncheck_all)}") %>
+              <%=h old_status.name %>
+            </td>
+            <% for new_status in @statuses -%>
+              <td>
+                <%= check_box_tag "status[#{ old_status.id }][#{new_status.id}][]", name, workflows.detect {|w| w.old_status_id == old_status.id && w.new_status_id == new_status.id},
+                :class => "old-status-#{old_status.id} new-status-#{new_status.id}" %>
+              </td>
+            <% end -%>
+          </tr>
+        <% end %>
+      </tbody>
+    </table>
+    <div class="generic-table--header-background"></div>
+  </div>
+</div>
```

Comment:
Any chance we can get a different styling for this one?

It is currently looking like this:
![image](https://cloud.githubusercontent.com/assets/617519/9355757/1c151efc-467c-11e5-8268-24e472773afb.png)

I'd rather have the status headings (checkmark + name of status) more visible as headings. To that end I would like:
* to have the column highlighting work on the the status heading
* to have the status heading in bold and moved below the stronger line

Don't know if it is possible though.